### PR TITLE
chore: improve mender convert docs

### PR DESCRIPTION
**Pull request by snusifer**
See commit messages for each item improved

### CODE REVIEW

**Review by oleorhagen — APPROVED** [review]
Good job!

**Comment by snusifer** [comment]
@lluiscampos might I get a second opinion/review on this, just in case?

File: 04.System-updates-Debian-family/02.Convert-a-Mender-Debian-image/docs.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ The easiest, and most straight-forward way, is to integrate the client with
 #### Using [hosted Mender](https://hosted.mender.io?target=_blank)
 ```bash
 mkdir -p input
-$MENDER_CONVERT_LOCATION/bootstrap-rootfs-overlay-hosted-server.sh \
+$MENDER_CONVERT_LOCATION/scripts/bootstrap-rootfs-overlay-hosted-server.sh \
```
Comment (lluiscampos) [inline 04.System-updates-Debian-family/02.Convert-a-Mender-Debian-image/docs.md:116]: You need to drop the first commit - it has been fixed (almost at the same time!) by Lucho at #1898

File: 04.System-updates-Debian-family/02.Convert-a-Mender-Debian-image/docs.md [hunk 2]
```diff
@@ -132,6 +132,14 @@ mv <PATH_TO_MY_GOLDEN_IMAGE> input/golden-image-1.img
...
 ### Use the mender-convert container image
 
+We strongly recommend using mender-convert with docker.
+
+Build the docker image:
+
+```bash
+./docker-build
+```
+
```
Comment (lluiscampos) [inline 04.System-updates-Debian-family/02.Convert-a-Mender-Debian-image/docs.md:142]: This part we should drop. Before we had to build the image but now we are publishing it so the recommended flow is just to pull it. See https://github.com/mendersoftware/mender-docs/commit/ddafd0a373b7499d2fac96a581a67069a31ec98d

File: 04.System-updates-Debian-family/02.Convert-a-Mender-Debian-image/01.Customization/docs.md [hunk 3]
```diff
@@ -184,3 +184,7 @@ MENDER_ARTIFACT_NAME=release-1 ./docker-mender-convert \
     --config configs/raspberrypi3_config \
     --overlay input/rootfs_overlay_demo/
 ```
+
+When passing multiple overlays mender-convert applies each overlay in the order
```
Comment (lluiscampos) [inline 04.System-updates-Debian-family/02.Convert-a-Mender-Debian-image/01.Customization/docs.md:188]:
```suggestion
When passing multiple overlays `mender-convert` applies each overlay in the order
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
See some comments below

> (...)There is also a point to make regarding [this pull](https://github.com/mendersoftware/mender-convert/pull/482). Perhaps we need some documentation regarding the bootstrap scripts themselves, and that one would need to pass it as an overlay to mender-convert?

As a follow-up, we could add such chapter, yes. I suggest we discuss it in Slack (mender-docs-team channel) and get the input from Customer Engineers.

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**